examples/Tile.props.tsx

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Tile, TagContainer, Tag, IconButton } from '@itwin/itwinui-react';
+import { SvgStar, SvgInfo } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <Tile
+      name='Tile Name'
+      thumbnail='https://itwinplatformcdn.azureedge.net/iTwinUI/stadium.png'
+      leftIcon={
+        <IconButton styleType='borderless' size='small' label='Favorite'>
+          <SvgStar />
+        </IconButton>
+      }
+      rightIcon={
+        <IconButton styleType='borderless' size='small' label='More info'>
+          <SvgInfo />
+        </IconButton>
```

Comment:
Should users use `Tile.IconButton` instead of `IconButton` in the Props API? I remember seeing L7 from #1527.

https://github.com/iTwin/iTwinUI/blob/a85e5ab81d8babc4bd09e02bbe00b3cea1693e02/.changeset/polite-walls-rush.md?plain=1#L5-L7

But I don't know if that was only for the Composition API or not. Because the way L7 is phrased, it made me think that requirement was for both APIs.


```suggestion
        <Tile.IconButton styleType='borderless' size='small' label='Favorite'>
          <SvgStar />
        </Tile.IconButton>
      }
      rightIcon={
        <Tile.IconButton styleType='borderless' size='small' label='More info'>
          <SvgInfo />
        </Tile.IconButton>
```

Reply:
It was decided to use IconButton and NOT mix APIs: https://github.com/iTwin/iTwinUI/pull/1527#discussion_r1310322354

Reply:
Ohh, then the changelog's L7 can be made a bit clearer.

Reply:
Updated in [1e36cc6](https://github.com/iTwin/iTwinUI/pull/1538/commits/1e36cc6890053baee9130f2b32eba3644f11cf03)